docs/root/intro/arch_overview/security/bug_bounty.rst

```diff
@@ -0,0 +1,176 @@
+.. _arch_overview_bug_bounty:
+
+Bug bounty
+==========
+
+In the near future, Envoy will have a bug bounty program. This will be open to all security
+researchers and will provide rewards for vulnerabilities discovered and reported according
+to the rules below.
+
+.. note::
+  This is not yet funded, stay tuned and reach out to envoy-security@googlegroups.com for further
+  details.
+
+.. _arch_overview_bug_bounty_rules:
+
+Rules
+-----
+
+The goal of this bug bounty is to provide a formal process to honor contributions from external
+security researchers to Envoy's security. Vulnerabilities should meet the following conditions
+to be eligible for the program:
+
+1. Vulnerabilities must meet one of the below :ref:`objectives
+   <arch_overview_bug_bounty_objectives>`, demonstrated with the supplied Docker-based
+   :ref:`execution environment <arch_overview_bug_bounty_ee>` and be consistent with the
+   program's :ref:`threat model <arch_overview_bug_bounty_threat_model>`.
+
+2. Vulnerabilities must be reported to envoy-security@googlegroups.com and be kept under embargo
+   while triage and potential security releases occur.
+
+3. Vulnerabilities must not be previously known in a public forum, e.g. GitHub issues trackers,
+   CVE databases (when previously associated with Envoy), etc. Existing CVEs that have not been
+   previously associated with an Envoy vulnerability are fair game.
+
+Rewards are at the discretion of the Envoy OSS security team and will be conditioned on the above
+criteria. If multiple instances of the same vulnerability are reported at the same time by
+independent researchers or the vulnerability is already tracked under embargo by the OSS Envoy
+security team, we will aim to fairly divide the reward amongst reporters.
+
+.. _arch_overview_bug_bounty_threat_model:
+
+Threat model
+------------
+
+The base threat model matches that of Envoy's :ref:`OSS security posture
+<arch_overview_threat_model>`. We add a number of temporary restrictions to provide a constrained
+attack surface for the initial stages of this program. We exclude any threat from:
+
+* Untrusted control planes.
+* Runtime services such as access logging, external authorization, etc.
+* Untrusted upstreams.
+* DoS attacks except as stipulated below.
+* Any filters apart from the HTTP connection manager network filter and HTTP router filter.
+* Admin console; this is disabled in the execution environment.
+
+In the future we will relax these restrictions as we increase the sophistication of the
```

Comment:
maybe move this below the next paragraph, as you might want to relax it in the future